### DISCUSSION TITLE
Can the comment of the database be attached when generating the client?

### DISCUSSION THREAD

**Discussion by MineYuanlu** [prisma/prisma#14333] [Q&A] [answered]
As the title goes, every `column` in the database can be attached with a `comment` (at least MySQL is like this)
Is it possible to bring this information into the client?

Current situation:
```ts
/**
 * Model user
 * 
 */
export type user = {
  id: number
  name: string | null
}
```
my sql:
```sql
CREATE TABLE `user` (
  `id` int NOT NULL AUTO_INCREMENT COMMENT 'the user id',
  `name` varchar(255) DEFAULT NULL COMMENT 'the user name',
) COMMENT='the user datas';
```
What I want:
```ts
/**
 * Model user
 * 
 * the user datas
 */
export type user = {
  /**the user id*/
  id: number
  /**the user name*/
  name: string | null
}
```
Even better, the definition of column can be attached to the comment:
```ts

  /**
   * the user id
   * 
   * ```sql
   * `id` int NOT NULL AUTO_INCREMENT COMMENT 'the user id'
   * ```
   */
```

**Comment by ludralph** [comment, marked as answer]
Hi @MineYuanlu 👋 

Prisma has support for comments on the schema level as can be seen from this [docs](https://www.prisma.io/docs/concepts/components/prisma-schema#comments)

At the moment Prisma does not have support for generating comments for the client. Also, triple slash comments would show up in the internal DMMF but won't show up in generated PrismaClient. However, you can open a feature [request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so our team can look into it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
